.github/pull_request_template.md

```diff
@@ -0,0 +1,56 @@
+# Description
+
+*Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.*
+
+## Related Issue(s)
+
+- Issue: #<issue_number_here>
+- See also:
+    - #<issue_number>
+
+## Type of change
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] Documentation
+- [ ] Other (please describe):
+
+# Changes
+
+## Added
+
+- 
+
+## Changed
+
+- 
+
+## Deprecated
+
+- 
+
+## Removed
+
+- 
+
+## Fixed
+
+- 
+
+## Security
+
+-
+
+# How has this been tested?
+
+*Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce.*
+
+# Checklist
+
+- [ ] My code follows the style guidelines of this project.
```

Comment:
Where are the style guidelines?

Reply:
That's the neat part. There are none. (Fixed.)